src/components/Scores.tsx

```diff
@@ -9,19 +9,22 @@ interface IProps {
 
 export default function Scores(props: IProps) {
     let playerStats: PlayerStats = props.playerStats;
+    let friends: number = Math.round(playerStats.getFriends() * 100) / 100;

```

Comment:
I thought we can just use `Math.round(playerStats.getFriends(), 2)` to round to 2 digits?

Reply:
Yeah that looks a bit cleaner